src/frontend/screens/BackgroundMaps/BackgroundMaps.tsx

```diff
@@ -265,35 +282,115 @@ export function BackgroundMapsScreen() {
   );
 }
 
-function CustomMapInfoSection({
+function NoMapScreen({
+  error,
   onChooseFile,
-  onRemoveMap,
+  onRemoveMapFile,
 }: {
+  error: Error | null;
   onChooseFile: () => void;
+  onRemoveMapFile: () => void;
+}) {
+  const {formatMessage: t} = useIntl();
+
+  return (
+    <>
+      <View style={styles.descriptionContainer}>
+        <BodyText>{t(m.description1)}</BodyText>
+        <BodyText>{t(m.description2)}</BodyText>
+      </View>
+      <ChooseMapFile onChooseFile={onChooseFile} />
```

Comment:
Were just prop forwarding here. I think we can just get rid of this `ChooseMapFile` component and copy and past it onto this file.

If your worried about how large this file is (which i personally don't think is a big deal) we could seperate the `NoMapScreen` and the `MapInfoScreen` into seperate files